netman/adapters/switches/brocade.py

```diff
@@ -515,8 +527,8 @@ def __init__(self, *args, **kwargs):
         super(VlanBrocade, self).__init__(*args, **kwargs)
 
         self.vlan_interface_name = kwargs.pop('vlan_interface_name', None)
-        self.tagged_interfaces = []
         self.icmp_redirects = True
+        self.has_no_untagged_interface = False
```

Comment:
I agree I was also confused by "has_no_untagged_interface". I feel we chose that name because brocade lists them as "no untagged". Can we instead of the boolean, add the untagged interfaces that we find to a list, called something like "untagged_interfaces" and then work with that data?